openchrom/plugins/net.openchrom.xxd.process.supplier.templates/src/net/openchrom/xxd/process/supplier/templates/peaks/PeakTransfer.java

```diff
@@ -272,7 +274,7 @@ private void transfer(IPeak peakSource, int startRetentionTime, int stopRetentio
 		//
 		IPeak peakSink = peakSupport.extractPeakByRetentionTime(chromatogramSink, startRetentionTime, stopRetentionTime, includeBackground, optimizeRange, traces);
 		if(peakSink != null) {
-			adjustPeakIntensity(peakSource, peakSink, percentageIntensity, peakTransferSettings);
+			adjustPeakIntensity(peakSource, percentageIntensity, peakTransferSettings);
```

Comment:
In this case, the intensity of the source peak is adjusted. I doubt, that this is done on purpose.

Reply:
Whoopsy. Well spotted.